AGENTS-GUIDE.md

```diff
@@ -0,0 +1,31 @@
+# Getting started with Nx Agents
+
+We recommend deploying Nx Agents onto a new cluster, but you can also deploy onto your existing NxCloud cluster under a different namespace.
+
+### Install Valkey
+
+Valkey is an in-memory DB (like Redis) that is used by the workflow controller to store temporary state.
+
+1. Create a secret similar to `agents-guide/agents-secrets.yml` and create a valkey password in there.
+   - Important: the key `valkey-password` needs to be remain unchanged
+   - You do not need to set the secret values for the S3 bucket yet. They are there as an example. Please refer to `charts/nx-agents/values.yaml` for examples on what Agent storage options we support.
+2. Apply the secret: `kubectl apply -f agents-secrets.yml`
+3. Now let's deploy Valkey:
+    ```bash
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+    helm install valkey bitnami/valkey --set auth.usePassword=true --set auth.existingSecret=nx-cloud-agents-secret
```

Comment:
@pmariglia were there any other options you used for deploying valkey that you think we should recommend here?